GOVERNANCE.md

```diff
@@ -65,109 +66,60 @@ including:
 * Conduct guidelines
 * Maintaining the list of additional Collaborators
 
-* [Current list of CTC members](./README.md#current-project-team-members)
+* [Current list of TSC members](./README.md#current-project-team-members)
 
-## CTC Membership
+The operations of the TSC are governed by the [TSC Charter][] as approved by
+the Node.js Foundation Board of Directors.
 
-CTC seats are not time-limited. There is no fixed size of the CTC. The CTC
-should be of such a size as to ensure adequate coverage of important areas of
-expertise balanced with the ability to make decisions efficiently.
+### TSC Meetings
 
-There is no specific set of requirements or qualifications for CTC
-membership beyond these rules.
-
-The CTC may add additional members to the CTC by a standard CTC motion.
-
-When a CTC member's participation in [CTC activities](#ctc-activities) has
-become minimal for a sustained period of time, the CTC will request that the
-member either indicate an intention to increase participation or voluntarily
-resign.
-
-CTC members may only be removed by voluntary resignation or through a standard
-CTC motion.
-
-Changes to CTC membership should be posted in the agenda, and may be
-suggested as any other agenda item (see [CTC Meetings](#ctc-meetings) below).
-
-No more than 1/3 of the CTC members may be affiliated with the same
-employer.  If removal or resignation of a CTC member, or a change of
-employment by a CTC member, creates a situation where more than 1/3 of
-the CTC membership shares an employer, then the situation must be
-immediately remedied by the resignation or removal of one or more CTC
-members affiliated with the over-represented employer(s).
-
-### CTC Activities
-
-Typical activities of a CTC member include:
-
-* attending the weekly meeting
-* commenting on the weekly CTC meeting issue and issues labeled `ctc-review`
-* participating in CTC email threads
-* volunteering for tasks that arise from CTC meetings and related discussions
-* other activities (beyond those typical of Collaborators) that facilitate the
-  smooth day-to-day operation of the Node.js project
-
-Note that CTC members are also Collaborators and therefore typically perform
-Collaborator activities as well.
-
-### CTC Meetings
-
-The CTC meets weekly in a voice conference call. The meeting is run by a
-designated meeting chair approved by the CTC. Each meeting is streamed on
+The TSC meets regularly in a voice conference call. The meeting is run by a
+designated meeting chair approved by the TSC. Each meeting is streamed on
 YouTube.
 
-Items are added to the CTC agenda which are considered contentious or
-are modifications of governance, contribution policy, CTC membership,
+Items are added to the TSC agenda which are considered contentious or
```

Comment:
This paragraph is now redundant because it contains information already included in the TSC Charter. It's still good to have it here, I think, but it may be worth including a sentence somewhere in this doc that notes that if anything written in this doc is in contradiction with something in the TSC Charter, the TSC Charter takes precedence.